hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonTableOutputFormat.java

```diff
@@ -267,9 +267,6 @@ public synchronized OutputCommitter getOutputCommitter(TaskAttemptContext contex
             iterator.closeWriter(true);
           }
           dataLoadExecutor.close();
-          // clean up the folders and files created locally for data load operation
```

Comment:
FYI: the clean operation will be done at the closure. there is no need to do this ahead